meditation/quantum-pause.md

```diff
@@ -39,10 +39,10 @@ found they help immensely.
 > As a meditator  
 > In order to start this process  
 > When I have a purpose in mind  
-> Then I declare it aloud, saying "$intent".
-    
+> Then I declare it aloud, saying "\$intent".
```

Comment:
$ is a special character in markdown? TIL

Reply:
Thank Prettier. I didn't know either!